client/my-sites/plans/current-plan/index.jsx

```diff
@@ -93,7 +92,12 @@ class CurrentPlan extends Component {
 		const currentPlanSlug = selectedSite.plan.product_slug,
 			isLoading = this.isLoading();
 
-		const { title, tagLine } = this.getHeaderWording( currentPlanSlug );
+		const planConstObj = getPlan( currentPlanSlug ),
+			planFeaturesHeader = translate( '%s plan features', {
```

Comment:
Please add a comment for the variable, for example change the it to `%(planName) plan features` and use `args: { planName: planConstObj.getTitle() }`. Thanks!

Reply:
Will do! Thanks.